clang/test/Analysis/cxx-uninitialized-object.cpp

```diff
@@ -1114,17 +1114,16 @@ void fCXX11MemberInitTest1() {
   CXX11MemberInitTest1();
 }
 
+#ifdef PEDANTIC
```

Comment:
I don't think you need to guard this section of code if you were using `// pedantic-note {{...}}` and `// pedantic-warning {{...}}` in the guarded checks.

I also think that the `// TODO: we'd expect the warning: {{2 uninitializeds field}}` comment refers to this new appearing report, so you should just drop that comment from the `fCXX11MemberInitTest2` function.

Reply:
Agree!

Reply:
done